test/ttmlir/Conversion/StableHLOToTTIR/unary_ops.mlir

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+module @jit_eltwise_abs attributes {} {
+  func.func public @test_abs(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
+    %0 = stablehlo.abs %arg0 : tensor<13x21x3xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
+    // CHECK: %[[C:.*]] = "ttir.abs"[[C:.*]]
+    return %0 : tensor<13x21x3xf32>
+  }
+
+  func.func public @test_ceil(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
+    %0 = stablehlo.ceil %arg0 : tensor<13x21x3xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
```

Comment:
Can you please improve these checks in tests that you've added? See this message: https://tenstorrent.slack.com/archives/C07A30HRX7C/p1727698795548319

Reply:
Thanks, StableHLO->TTIR tests look good now, can you please do the same for TTIR->TTNN tests you've added? No need to fix silicon tests, I'm not even sure why do they have checks, probably because of copy paste from TTNN tests.